proto/aserto/directory/schema/v2/tenant.proto

```diff
@@ -48,3 +51,12 @@ message GuideState {
   // Progress information about individual steps in the guide.
   google.protobuf.Struct steps = 2;
 }
+
+
+// The state of a user's progress through the console's onboarding process.
+message OnboardingState {
```

Comment:
Would it make sense to introduce some kind of versioning for the onbroading process itself?
The current version would be 1 and if we change the onboarding process in the future, we'll be able to tell if a user started onboarding using a previous version and never completed it.
In that case we may want to restart the onboarding process for that account.

Reply:
I think that would assume you can rely on the state that a tenant would be in, not clear to me this is the case. If the user finished onboarding, they can destruct it the minute after, unless I am missing something.
If not, we probably would need to know the onboarding flavor, and the expected outcome state, so you can validate this, none of that make a ton of sense to me nor feels like something one can rely on to be true.